### PR TITLE
Hint edits

### PR DESCRIPTION
**Pull request by Gschillereff**

### CODE REVIEW

File: app/src/main/java/com/neongarage/CountInActivity.java [hunk 1]
```diff
@@ -32,13 +39,13 @@ public double getTotal() {
...
-    EditText nickelsEdit;
-    EditText dimesEdit;
-    EditText quarterEdit;
-    EditText onesEdit;
-    EditText fivesEdit;
-    EditText tensEdit;
-    EditText twentiesEdit;
+    TextInputEditText nickelsEdit;
+    TextInputEditText dimesEdit;
+    TextInputEditText quartersEdit;
```
Comment (Ace4) [inline app/src/main/java/com/neongarage/CountInActivity.java:44]: sssssss

File: app/src/main/res/layout/count_in_activity.xml [hunk 2]
```diff
@@ -25,116 +24,179 @@
         android:layout_height="wrap_content"
         android:hint="Coin Total"
         app:layout_constraintBottom_toTopOf="@+id/dollar_view"
-        app:layout_constraintEnd_toStartOf="@+id/twentites_count"
```
Comment (Ace4) [inline app/src/main/res/layout/count_in_activity.xml]: 8)

File: app/src/main/res/layout/count_in_activity.xml [hunk 3]
```diff
@@ -25,116 +24,179 @@
...
-    <EditText
+
+    <com.google.android.material.textfield.TextInputLayout
         android:id="@+id/fives_count"
         android:layout_width="0dp"
         android:layout_height="wrap_content"
-        android:ems="10"
         android:hint="@string/fives"
-        android:inputType="number"
-        android:textSize="16sp"
```
Comment (Ace4) [inline app/src/main/res/layout/count_in_activity.xml:39]: Why remove textsize?

Comment (Ace4) [reply app/src/main/res/layout/count_in_activity.xml:39]: You can add this to the TextInputEditText if you want

**Review by Ace4 — APPROVED** [review]
👍

File: app/src/main/res/layout/count_in_activity.xml (hunk 3 above)
Comment (Gschillereff) [reply app/src/main/res/layout/count_in_activity.xml:39]: "But now it will default to a reasonable size"
- Aaron Egger

**Pull request merged**